diff --git a/build/devtools.js b/build/devtools.js
index 10afb3a8e8c7ba1c66e962e57a708586bd5cc5cb..df505aa069eabd63b268cf6eb844c712dcfc3783 100644
--- a/build/devtools.js
+++ b/build/devtools.js
@@ -1,11 +1,2 @@
-/* eslint-disable import/order */
-// eslint-disable-next-line import/no-unassigned-import
-import './devtools-window-polyfill.js';
-// eslint-disable-next-line @typescript-eslint/ban-ts-comment
-// @ts-expect-error
-import devtools from 'react-devtools-core';
-// eslint-disable-next-line @typescript-eslint/no-unsafe-call
-devtools.initialize();
-// eslint-disable-next-line @typescript-eslint/no-unsafe-call
-devtools.connectToDevTools();
-//# sourceMappingURL=devtools.js.map
\ No newline at end of file
+// Stubbed out for bundling — react-devtools-core is an optional dev dependency.
+export {};
